ecl/sss/v1/approval_requests/testing/fixtures.go

```diff
@@ -9,72 +9,101 @@ import (
 const idApprovalRequest1 = "9a76dca6-d8cd-4391-aac6f-2ea052f10f4"
 const idApprovalRequest2 = "fc578e8b-dea2-4f8c-aa7e-9026fa173632"
 
-const startTime = "2018-07-26 08:40:01"
+var actionsBody = []byte(
```

Comment:
It looks mess to me.
```suggestion
var actionsBody = json.RawMessage(fmt.Sprint(
	`{`,
	`"firewall":`,
	`{`,
	`"availability_zone":"zone1-groupa",`,
	`"default_gateway":"",`,
	`"description":"abcdefghijklmnopqrstuvwxyz",`,
	`"firewall_plan_id":"bd12784a-c66e-4f13-9f72-5143d64762b6",`,
	`"name":"abcdefghijklmnopqrstuvwxyz",`,
	`"tenant_id":"6a156ddf2ecd497ca786ff2da6df5aa8"`,
	`}`,
	`}`,
))
```

Reply:
My understanding is that API resp is byte before extracting, but not 100% sure..

Reply:
Actions.Body contains json.RawMessage, so do the same for actionsBody.